packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -87,7 +88,9 @@ export const makeSelectProps = propsFactory({
   openOnClear: Boolean,
   itemColor: String,
 
-  ...makeItemsProps({ itemChildren: false }),
+  ...makeItemsProps({
+    itemProps: () => (item: any) => ({ subheader: item.subheader, divider: item.divider }),
```

Comment:
I tested the branch with the provided markup and the dividers are missing for the VSelect (and VAutocomplete). It seems like this is expecting `subheader` and `divider` to be at the root, but it is not the case.

![image](https://github.com/vuetifyjs/vuetify/assets/101416921/86b819e8-b55e-4f6e-8b89-87d818b105ec)


Looking into `item.props?.subheader` works. It also works like that:

```suggestion
    itemProps: 'props',
```

![image](https://github.com/vuetifyjs/vuetify/assets/101416921/b5f8348e-aaf2-4130-a0b1-7c1536d51e4a)

Reply:
Unless the markup you provided is not the good one @func0der ? I see on the [original PR](https://github.com/vuetifyjs/vuetify/pull/15728) that `subheader` and `divider` were at the root of the item

Reply:
Thanks for the pointer. 
I could not make out any occurence where this change would be needed.
I guess, it is just a leftover from the time this features was experimented with.

I changed this to have only the `itemChildren` removed.

Reply:
It looks good with your change @func0der, thank you !